ihmc-high-level-behaviors/src/main/java/us/ihmc/behaviors/activeMapping/ContinuousPlanner.java

```diff
@@ -326,6 +330,21 @@ public void setGoalWaypointPoses()
                                                                    (float) continuousHikingParameters.getGoalPoseForwardDistance(),
                                                                    xRandomMargin,
                                                                    (float) continuousHikingParameters.getGoalPoseUpDistance(), nominalStanceWidth);
+         case WALK_TO_GOAL:
```

Comment:
Allows for Continuous Hiking to walk to a goal as well

Reply:
do you know what frontier expansion and active search modes do?

Reply:
Are you able to change the mode via the API? Is the intention that the mode is assumed to get set on construction and never change? Otherwise, it seems like the mode could dynamically change based on the user's request.

Reply:
I don't know about changing the mode via the API